clang/lib/AST/ASTContext.cpp

```diff
@@ -14296,6 +14296,29 @@ void ASTContext::getFunctionFeatureMap(llvm::StringMap<bool> &FeatureMap,
   }
 }
 
+static SYCLKernelInfo BuildSYCLKernelInfo(CanQualType KernelNameType,
```

Comment:
I agree it is silly as is. It will get bigger; see [here](https://github.com/tahonermann/llvm-project/compare/sycl-upstream-fe-pr3...tahonermann:llvm-project:sycl-upstream-fe-pr4#diff-328fcdcf882c121d5c56fdc6cd56c43fdbbb00da171da70f3ba6982c43686a63R14315-R14357).

Reply:
Got it, thanks!